### PR TITLE
#241 and #522

### PR DESCRIPTION
**Pull request by ake123**
New citation #522
Add references #241
Permanent solution to polyfill issue by disabling mathjax

### CODE REVIEW

File: inst/assets/bibliography.bib [hunk 1]
```diff
@@ -1,4 +1,22 @@
 
+@Article{Xu2023,
+  author =	 {Shuangbin Xu and Li Zhan and Wenli Tang and Qianwen Wang and Zehan Dai and Lang Zhou and Tingze Feng and Meijun Chen and Tianzhi Wu and Erqiang Hu and Guangchuang Yu},
+  title =	 {MicrobiotaProcess: A comprehensive R package for deep mining microbiome},
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: Add double curly brackets around words where capitalization matters (always)

File: inst/assets/bibliography.bib [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+  journal =	 {The Innovation},
+  year =	 {2023},
+  volume =	 4,
+  doi =		 {10.1016/j.xinn.2023.100388}
+}
+
+@Article{Deng2024,
+  author =	 {Yongcui Deng and Alexander K Umbach and Josh D Neufeld},
+  title =	 {Nonparametric richness estimators Chao1 and ACE must not be used with amplicon sequence variant data},
+  journal =	 {The ISME Journal},
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: same

File: inst/assets/bibliography.bib [hunk 3]
```diff
@@ -1,4 +1,22 @@
...
+  title =	 {MicrobiotaProcess: A comprehensive R package for deep mining microbiome},
+  journal =	 {The Innovation},
+  year =	 {2023},
+  volume =	 4,
+  doi =		 {10.1016/j.xinn.2023.100388}
+}
+
+@Article{Deng2024,
+  author =	 {Yongcui Deng and Alexander K Umbach and Josh D Neufeld},
+  title =	 {Nonparametric richness estimators Chao1 and ACE must not be used with amplicon sequence variant data},
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: same

File: inst/assets/bibliography.bib [hunk 4]
```diff
@@ -1,4 +1,22 @@
...
+  journal =	 {The ISME Journal},
+  year =	 {2024},
+  volume =	 18,
+  doi =		 {10.1093/ismejo/wrae106}
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: is there page info available?

File: inst/assets/bibliography.bib [hunk 5]
```diff
@@ -1,4 +1,22 @@
 
+@Article{Xu2023,
+  author =	 {Shuangbin Xu and Li Zhan and Wenli Tang and Qianwen Wang and Zehan Dai and Lang Zhou and Tingze Feng and Meijun Chen and Tianzhi Wu and Erqiang Hu and Guangchuang Yu},
+  title =	 {MicrobiotaProcess: A comprehensive R package for deep mining microbiome},
+  journal =	 {The Innovation},
+  year =	 {2023},
+  volume =	 4,
+  doi =		 {10.1016/j.xinn.2023.100388}
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: is there page info available?

File: inst/pages/acknowledgments.qmd [hunk 6]
```diff
@@ -98,21 +98,21 @@ in writing the chapters on network learning and comparison.
...
+- *Pande Erawijantari*
 
--*Shadman Ishraq*
+- *Shadman Ishraq*
 
--*Sam Hillman*
+- *Sam Hillman*
 
--*Matteo Calgaro*
+- *Matteo Calgaro*
```
Comment (antagomir) [inline inst/pages/acknowledgments.qmd]: Could you check that the acknowledgment list contains all who have committed to mia, miaViz, miaSim, miaTime, OMA?

**Review by antagomir — COMMENTED** [review]
some suggested updates.

**Comment by antagomir** [comment]
Resolve the conversations once done + resolve the conflicts.

File: inst/assets/bibliography.bib (hunk 5 above)
Comment (ake123) [reply inst/assets/bibliography.bib]: added!

File: inst/pages/acknowledgments.qmd (hunk 6 above)
Comment (ake123) [reply inst/pages/acknowledgments.qmd]: Yes Done!

**Comment by antagomir** [comment]
Resolve the conversations once done

**Pull request merged**